packages/medusa/src/services/claim.js

```diff
@@ -230,7 +230,35 @@ class ClaimService extends BaseService {
       let toRefund = refund_amount
       if (type === "refund" && typeof refund_amount === "undefined") {
         const lines = claim_items.map((ci) => {
-          const orderItem = order.items.find((oi) => oi.id === ci.item_id)
+          const allOrderItems = order.items
+
+          if (order.swaps && order.swaps.length) {
```

Comment:
very nit(non-blocking):
```suggestion
          if (order?.swaps?.length) {
```